app/src/ActionModal/components/ModalDeposit.vue

```diff
@@ -175,7 +158,7 @@ export default {
   methods: {
     open() {
       if (this.currentNetwork.network_type === `polkadot`) {
-        this.amount = this.minimumDeposit
+        this.amount = this.deposits[0].amount[0].amount
```

Comment:
well, this also makes sense, true. Actually it would be always the prior deposit, that is, the last one in the Array (`this.deposits[0].amount[this.deposits.length - 1].amount`)

Reply:
Ah yeah, but logically must be all the same :sweat_smile: